frontend/src/app/components/filters/query-filter/query-filter.component.ts

```diff
@@ -34,6 +34,7 @@ import {QueryFilterInstanceResource} from "core-app/modules/hal/resources/query-
 import {BannersService} from "core-app/modules/common/enterprise/banners.service";
 import {WorkPackageViewFiltersService} from "core-app/modules/work_packages/routing/wp-view-base/view-services/wp-view-filters.service";
 import {SchemaCacheService} from "core-components/schemas/schema-cache.service";
+import { CurrentProjectService } from 'core-app/components/projects/current-project.service';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/363f7cab35d338fab8e49ffc6a6df62340a60593/frontend/src/app/components/filters/query-filter/query-filter.component.ts#L38"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>